### PR TITLE
Add examples/cli.sh

### PR DESCRIPTION
**Pull request by Tagar**
Haven't found a wrapper for very frequently used utilities in 
`io.druid.cli.Main `

### CODE REVIEW

**Comment by jihoonson** [comment]
Hi @Tagar, thanks for the contribution! Would you elaborate more on what this script is for?

**Comment by Tagar** [comment]
Some of the use cases 

`./cli.sh tools pull-deps --hadoop-coordinate org.apache.hadoop:hadoop-client:2.6.5`

or 

`./cli.sh index hadoop hadoopDependencies org.apache.hadoop:hadoop-client:2.6.5 mv-parquet-hadoop.json
`

File: examples/bin/cli.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+java -Xmx256m -Duser.timezone=UTC -Dfile.encoding=UTF-8 \
+        -cp lib/*:config/_common:config/broker \
```
Comment (jihoonson) [inline examples/bin/cli.sh]: Would you tell me why it includes the broker configuration?

Comment (Tagar) [reply examples/bin/cli.sh]: corrected

File: examples/bin/cli.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+java -Xmx256m -Duser.timezone=UTC -Dfile.encoding=UTF-8 \
+        -cp lib/*:config/_common \
+        io.druid.cli.Main $* \
+                | grep -v " INFO \[main\] io.druid.initialization.Initialization "
```
Comment (jihoonson) [inline examples/bin/cli.sh]: Would you tell me what `grep` is for?

Comment (gianm) [reply examples/bin/cli.sh]: Would probably be nicer to do this via a log4j2-cli.xml designed for the CLI tools.

**Comment by jihoonson** [comment]
Hi @Tagar, would you please check @gianm's last comment (https://github.com/apache/incubator-druid/pull/6672#discussion_r238850966)?

File: examples/bin/cli.sh (hunk 2 above)
Comment (Tagar) [reply examples/bin/cli.sh]: @gianm I agree that makes sense. It will look cleaner.

**Comment by Tagar** [comment]
@jihoonson I might have a look at this one later, once I have time.. 
Thanks!

**Comment by jihoonson** [comment]
Thanks @Tagar. I'm closing this PR now and please reopen it when you have time to continue.

**Pull request closed without merging**